simulation/include/traccc/simulation/smearing_writer.hpp

```diff
@@ -73,7 +73,7 @@ struct smearing_writer : detray::actor {
         hit_writer m_hit_writer;
         measurement_writer m_meas_writer;
         measurement_hit_id_writer m_measurement_hit_id_writer;
-        uint64_t m_hit_count = 0u;
+        measurement_id_type m_hit_count = 0u;
```

Comment:
🤔 Does this variable need to change type? I thought it was a "count". It looks independent from the measurement ID, no?